plugins/nordic-hid/fu-nordic-hid-cfg-channel.c

```diff
@@ -1294,19 +1297,14 @@ fu_nordic_hid_cfg_channel_set_quirk_kv(FuDevice *device,
 	FuNordicHidCfgChannel *self = FU_NORDIC_HID_CFG_CHANNEL(device);
 
 	if (g_strcmp0(key, "NordicHidBootloader") == 0) {
-		if (g_strcmp0(value, "B0") == 0)
-			self->bl_name = g_strdup("B0");
-		else if (g_strcmp0(value, "MCUBOOT") == 0)
-			self->bl_name = g_strdup("MCUBOOT");
-		else if (g_strcmp0(value, "MCUBOOT+XIP") == 0)
-			self->bl_name = g_strdup("MCUBOOT+XIP");
-		else {
+		if (g_strcmp0(value, "B0") != 0) {
 			g_set_error_literal(error,
 					    G_IO_ERROR,
 					    G_IO_ERROR_INVALID_DATA,
-					    "must be 'B0', 'MCUBOOT' or 'MCUBOOT+XIP'");
+					    "can be only 'B0' in quirk");
 			return FALSE;
 		}
+		self->bl_name = g_strdup("B0");
```

Comment:
maybe `g_strdup(value);` for future proofing?

Reply:
Changed. Also updated generation handling for consistency